app/uk/gov/hmrc/bindingtariffclassification/model/Search.scala

```diff
@@ -35,6 +36,7 @@ case class Search
 
 case class Filter
 (
+  applicationType: Option[ApplicationType] = None,
```

Comment:
I've had to add Application Type filter as the UI does it all over the place. With pagination and result filtering it would look all weird as we would say "Showing 20 results" and there would actually be less than that (as Liabilities would be dropped)